blocks/leadspace/leadspace.css

```diff
@@ -541,7 +539,7 @@ main div.leadspace-wrapper {
   .leadspace.video .video-control {
     bottom: 60px;
     right: 0;
-    margin-right: -55px;
+    margin-right: -32px;
```

Comment:
Why did you change this? The pause button actually hangs outside their grid per their design. 
![image](https://user-images.githubusercontent.com/3883795/236010725-d1ec6f92-1594-425b-97f8-6dce4291aa85.png)